Agenda Maker/README.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Agenda Maker for Google Calendar
+description: Creates an agenda for a calendar event you create automatically
+labels: Apps Script,  Calendar
+material_icon: event
+create_time: 2020-07-19
+update_time: 2020-07-19
+---
+
+Contributed by Jeremy Glassenberg [Site](https://www.apistrategist.com)
+
+So many meetings are time lost. No agenda means no one is prepared to discuss. No minutes means no one will remember what was decided. Most companies ask everyone to set agendas for their meetings, but can't create a good habit.
+
+It's just too burdensome to create an agenda, but very easy to create a meeting. With Agenda maker, the process of creating an agenda is much easier.
+
+By simply adding "#agenda" to the description of your Google Calendar invite, an agenda will be created for your meeting, with a Google Docs reminder to fill it in.
+
+Design Details:
+
+
+![demo](AgendaMakerScreenshot.png)
+
+## Technology highlights
+
+- Uses [Apps Script][apps_script], allowing for rapid
+  development and worry-free operation.
+- Uses the [Google Docs API][gdocs_api] to create a new Google Doc from the Agenda Template
+- Uses the [Google Drive API][gdrive_api] to set up an agenda folder
+- Uses the [Calendar API][calendar_api] to detect new events and update the event description with a link to the Agenda.
+
+[app_script]: https://developers.google.com/apps-script/
+[gdrive_api]: https://developers.google.com/drive/api/
+[gdocs_api]: https://developers.google.com/docs/api/
+[calendar_api]: https://developers.google.com/calendar/
+
+## Try it
+
+First, create the script:
+
+1.  [Click here][code] to open the code in the Apps Script code editor.
+2.  Click the menu item **File > Make a copy** to get your own copy. You can
+    later access this copy in the [G Suite Developer Hub][hub].
+
+[code]: https://github.com/gsuitedevs/solutions/tree/master/Agenda%20Maker/src/code.js
```

Comment:
apologies in advance for being a sticker but can you capitalize Code.js like the in the template? I can't remember if our automation expects but better to be safe than sorry!